backend/requirements/base.txt

```diff
@@ -6,7 +6,8 @@ djangorestframework-camel-case==1.0b1
 djangorestframework-jwt==1.11.0
 djangorestframework==3.9.1
 Markdown==3.0.1
-psycopg2==2.7.5
 Pygments==2.3.1
 django-countries==5.3.3
 django-import-export==2.0.2
+serverless-wsgi==1.7.4
```

Comment:
are these changes required to bump python?

Reply:
nope, that slipped trough from other pull request